roleData.js

```diff
@@ -3,113 +3,113 @@ const roleData = {
     {
       "title": "The Blood Empress",
       "ability": "(Start the game with this identity face up in the command zone. You are the starting player.)Whenever one or more creatures an opponent controls die, you may pay {1}. If you do, choose one —• Create a 1/1 colorless Spirit creature token.• Tap or untap target nonland permanent.• Each opponent loses 2 life and you gain 2 life.• Scry 2."
-    },
+      , "url": "https://imgur.com/0SrVoVB" },
```

Comment:
This is fine for now, but we're going to want to move these to our codebase since the vercel deploy is free.